mne/viz/_brain/tests/test_brain.py

```diff
@@ -1209,6 +1210,37 @@ def test_calculate_lut():
         calculate_lut(colormap, alpha, 1, 0, 2)
 
 
+def test_brain_ui_events():
```

Comment:
This needs something like
```suggestion
def test_brain_ui_events(renderer_interactive_pyvistaqt, brain_gc):
```